schemas/task.yml

```diff
@@ -13,6 +13,14 @@ properties:
     minLength:      {$const: identifier-min-length}
     maxLength:      {$const: identifier-max-length}
     pattern:        {$const: identifier-pattern}
+  clientId:
+    title:          "Client Id"
+    description: |
+      Unique identifier for a client, who creates the task
+    type:           string
+    minLength:      {$const: identifier-min-length}
+    maxLength:      {$const: identifier-max-length}
+    pattern:        {$const: identifier-pattern}
```

Comment:
ClientIds have a different schema than other identifiers: `pattern:  "^[A-Za-z0-9!@/:.+|_-]+$"`.  There's no minimum or maximum length.

I forgot to mention this in the last review, sorry.